vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_w8a8_int8.py

```diff
@@ -44,8 +46,21 @@ def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
         if self.is_static_input_scheme:
             layer.input_scale = Parameter(layer.input_scale.max(),
                                           requires_grad=False)
+            if not self.input_symmetric:
```

Comment:
I dont think this is needed

Reply:
Hmm interesting, yeah not sure how to adapt this for asymmetric, would probably have to reconstruct the ranges here

Reply:
This case is actually not necessary if we don't support the static per-tensor case